example-app/ios/App/Podfile

```diff
@@ -0,0 +1,28 @@
+require_relative '../../../node_modules/@capacitor/ios/scripts/pods_helpers'
+
+platform :ios, '13.0'
```

Comment:
Looking at the plugin, we aim to have a minimum version of iOS 14. Shouldn't this match that requirement?